PointPrim.cpp

```diff
@@ -27,14 +27,6 @@
 
 BOOL __fastcall Interpreter::primitiveMakePoint(CompiledMethod&, unsigned argCount)
 {
-#ifdef TIMEDEXPIRY
```

Comment:
Right. I did figure out that TIMEDEXPIRY was associated. What I missed was the ImageStamp. And I'm still trying to figure out why the tests are failing on the server since they pass for me locally.